accepted/2020/platform-exclusion/platform-exclusion.md

```diff
@@ -0,0 +1,217 @@
+# Annotating APIs as unsupported on specific platforms
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+For .NET 5, we're building a feature which [annotates platform-specific
+APIs][platform-checks]. The goal is to provide warnings when developers
+accidentally call platform-specific APIs, thus limiting their ability to run on
+multiple platforms, or if the code might run on earlier platform that doesn't
+support the called API yet.
+
+This feature made the following assumption:
+
+* An unmarked API is considered to work on all OS platforms.
+* An API marked with `[MinimumOSPlatform("...")]` is considered only portable to
+  the specified OS platforms (please note that the attribute can be applied
+  multiple times).
+
+This approach works well for cases where the API is really calling an
+OS-specific API. Good examples include the Windows Registry or Apple-specific
+graphic APIs. This is very different from browser support for specific HTML and
+CSS features: those generally aren't browser specific, which is why browser
+checks never worked well for the web. But for OS specific APIs, asking whether
+you're actually running on that OS is very workable. The only fragile aspect
+here is getting the version number right, but the analyzer enforces that.
+
+However, this approach doesn't work well for APIs that represent concepts that
+aren't OS specific, such as threading or I/O. This is a problem for OS platforms
+that are more constrained, such as Blazor Web Assembly, which runs in a browser
```

Comment:
use WebAssembly without space (applies throughout the doc).

Reply:
Yessir